docs/how-to-authenticate-private-repositories.md

```diff
@@ -1,6 +1,6 @@
-# How To Authenticate Private App-Registry Repositories
+# How To Authenticate Private registry Repositories
 
-If you have an app-registry repository that is backed by authentication, you can specify an authentication token in a Secret. To do this, create a Secret in the same namespace as your Operator Source:
+If you have an registry repository that is backed by authentication, you can specify an authentication token in a Secret. To do this, create a Secret in the same namespace as your Catalog Source:
```

Comment:
This needs to be omitted too since private repo authentication is not possible using CatalogSource